### ISSUE TITLE
Awesome Windows Repository

### ISSUE THREAD

**Issue by rahulkapoor90** [RiseLedger/awesome-windows#29]
@RiseLedger what path shall we take?

Ref - [https://github.com/sindresorhus/awesome/issues/671](url)

**Comment by egeerardyn** [comment]
@rahulkapoor90 It seems something odd is going on with your links.
For convenience a link back to the main repository. https://github.com/sindresorhus/awesome/issues/671

In general, I think there are a few things you guys could try to sort out:
 - make sure you mention something like `This list was inspired by [awesome-OSX](https://github.com/iCHAIT/awesome-osx).` just for licensing compliance, if that is indeed the list you looked at.
 - decide on a common structure, what things can stay, what things should go, ...
 - maybe create an [organization](https://github.com/blog/674-introducing-organizations) that both of you own, if ownership of the "original"/"one true" repository is a problem.

**Comment by RiseLedger** [comment]
@egeerardyn Thanks for suggestion. Yes indeed the title and some categories at my side was inspired from `[awesome-OSX]` list. I have added a thanks section to my list. If something else is needed let me know.

@rahulkapoor90 About the path that we can take, I think we can merge our list into one, and exclude what is not necessary. From what I see there is 3 options:

1. Merge your list into my, and exclude what is not necessary, and you add a reference to your list that it will be removed in a month, and forward all users to my list.
2. Merge my list into your, and exclude what is not necessary, and I add a reference to my list that it will be removed in a month, and forward all users to your list.
3. You purpose an option that fit best for you

**Comment by rahulkapoor90** [comment]
@RiseLedger You could remove your's and add a reference to my list but I also have this feeling that i don't want neither of our repositories to be removed in preference of another as it may disappoint the one whose repository is going to be removed **instead** what i propose as @egeerardyn mentioned we could create an **organization** with the awesome windows repository in it and we collaborate together and further decide on a common structure, what things can stay, what things should go and really make this a much better list than it certainly exists and if things go well we may also add more **awesome** lists in that organization.
What do you say @RiseLedger ?

**Comment by RiseLedger** [comment]
@rahulkapoor90 Yes, this sound like the best approach. I never work with **organization** process before, so if you know how to make it correctly, go ahead. Just let me know what is needed from my side.

Thanks

**Comment by rahulkapoor90** [comment]
@RiseLedger Okay great! I have made the **Organization** and i have added so now you have to accept my invitation. There i created the initial repository, now what you and i will do first is to do the following:

1. Add the reference of the new one in our own existing individual lists and start diverting users.
2. Add the message that we will be removing the list in 10 days.
3. @RiseLedger you first tell me the common structure properly and we should start making the list.
4. After few weeks, we will send the pull request to update the main Awesome repository by sindresorhus.

Sounds good?

**Comment by RiseLedger** [comment]
@rahulkapoor90 I have join the **Organization**. Please also change my role to Owner.

About the next steps:

1. Ok
2. I think we should give more time, like 1 month for example
3. About the structure I think we can create one based on our both list
4. Ok

**Comment by rahulkapoor90** [comment]
@RiseLedger Changed your role to Owner, 1 month is fine and i agree about the structure. Can you  push the initial commit ?

**Comment by RiseLedger** [comment]
I have add basically all the list that are now in my repository. Feel free to play with it, adjust and add your also. I have not added yet contribution guide, and license. So choose the license which better fit, and for contribution guide we can adjust together.

**Comment by rahulkapoor90** [comment]
Yes i saw it and have you added the message in your own repository?

**Comment by RiseLedger** [comment]
Yes, take a look at the top of list. I think this message should be fine.

**Comment by rahulkapoor90** [comment]
@RiseLedger I have deleted my own repository, i hope to see you doing the same soon as we discussed and you haven't been active and contributing to main repository, may i know why?

**Comment by RiseLedger** [comment]
Hi @rahulkapoor90 , sure I will delete today the repo also. I just not in the country right now, and will be at the end of month. 

Thanks and great job with contribution.

**Comment by aloisdg** [comment]
You could come to the [awesome organization](https://github.com/awesome-lists).
